client/tasks/fills/Marketing/index.tsx

```diff
@@ -207,6 +207,7 @@ const Marketing: React.FC = () => {
 					</CardHeader>
 					<PluginList
 						currentPlugin={ currentPlugin }
+						installAndActivate={ installAndActivate }
```

Comment:
If this PR will get cherry picked we should add this change to here as well -> https://github.com/woocommerce/woocommerce-admin/blob/main/client/task-list/tasks/Marketing/index.tsx#L191-L194

Reply:
Good catch!